core/blockchain/block_tree.hpp

```diff
@@ -6,11 +6,12 @@
 #ifndef KAGOME_BLOCK_TREE_HPP
 #define KAGOME_BLOCK_TREE_HPP
 
+#include <boost/optional.hpp>
 #include <cstdint>
 #include <vector>
 
```

Comment:
```suggestion
#include <cstdint>
#include <vector>

#include <boost/optional.hpp>
```

Reply:
Why? Have we decided to separate the includes of different libraries?